### PR TITLE
feat: enable benchmark on organizational management account

### PR DESCRIPTION
**Pull request by aq17**
Previously, the organizational example did not create a cloud account and include the management account in the benchmark task scope

### CODE REVIEW

File: modules/services/cloud-bench/variables.tf [hunk 1]
```diff
@@ -34,3 +34,9 @@ variable "tags" {
     "product" = "sysdig-secure-for-cloud"
   }
 }
+
+variable "provision_in_management_account" {
+  type        = bool
+  default     = true
+  description = "Whether to deploy the stack in the management account"
```
Comment (wideawakening) [inline modules/services/cloud-bench/variables.tf:41]: guess we can hold exposing this variable to the examples

**Comment by wideawakening** [comment]
doing some maintenance on module so will add the "cft stackset isntance" to managed account on diagrams too

File: modules/services/cloud-bench/variables.tf (hunk 1 above)
Comment (aq17) [reply modules/services/cloud-bench/variables.tf:41]: 👍🏼

**Review by nkraemer-sysdig — APPROVED** [review]
This looks great, thanks @aq17!

**Comment by wideawakening** [comment]
i was iterating this yesterday and got 409 cloud-account setup conflict over and over again thinking it may be due to other causes, but after properly passing checks on other PR's (with some cleanup).. i'm thinking there may be something wrong with these changes.

been digging on the  `organizations_organization` data and seems like you got it all right, but dunno what's making it fail. could 
you give a look see if you spot what's going on?
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/organizations_organization#master-account-attributes-reference

---

some non-critical code review
- per terraform-guideliness, there should only be one `local` block, positioned on the top of the manifest
- `account_ids_to_deploy` seems like `concat(local.member_account_ids, [data.aws_organizations_organization.org[0].master_account_id])`

**Comment by aq17** [comment]
> i was iterating this yesterday and got 409 cloud-account setup conflict over and over again thinking it may be due to other causes, but after properly passing checks on other PR's (with some cleanup).. i'm thinking there may be something wrong with these changes.
> 
> been digging on the `organizations_organization` data and seems like you got it all right, but dunno what's making it fail. could you give a look see if you spot what's going on? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/organizations_organization#master-account-attributes-reference
> 
> some non-critical code review
> 
> * per terraform-guideliness, there should only be one `local` block, positioned on the top of the manifest
> * `account_ids_to_deploy` seems like `concat(local.member_account_ids, [data.aws_organizations_organization.org[0].master_account_id])`

Is the 409 for all accounts or specifically the master account?

**Comment by wideawakening** [comment]
[fix for tests through the provider ](https://github.com/sysdiglabs/terraform-provider-sysdig/pull/152) running smoothly
thx for the work @aq17!

**Pull request merged**